src/flows/data_retention/deleted_accounts_flow.py

```diff
@@ -1,46 +1,47 @@
-from prefect import task, Flow, Parameter
+from prefect import Flow, task
+from prefect.tasks.snowflake import SnowflakeQueriesFromFile
 
-from api_clients.pocket_snowflake_query import PocketSnowflakeQuery
-from utils.flow import get_flow_name, get_cron_schedule
-from utils.config import SNOWFLAKE_DATA_RETENTION_CONNECTION_DICT, SNOWFLAKE_DATA_RETENTION_DB, SNOWFLAKE_DATA_RETENTION_SCHEMA
+from utils import config
+from utils.flow import get_flow_name, get_cron_schedule, get_flow_file_path
 
 FLOW_NAME = get_flow_name(__file__)
 
-# Add new deleted user accounts
-ADD_DELETED_USERS_SQL = """
-insert into DELETED_USERS
-    (hashed_user_id, user_id)
-select distinct a.hashed_user_id, a.user_id
-from analytics.dbt_staging.stg_account_deletions as a
-left join DELETED_USERS as b 
-    on b.hashed_user_id = a.hashed_user_id and b.user_id = a.user_id
-where b.hashed_user_id is null;
-"""
-
-# Add new deleted user emails
-ADD_DELETED_EMAILS_SQL = """
-insert into DELETED_EMAILS
-    (email)
-select distinct m.email
-from analytics.dbt_staging.stg_user_to_email_map as m
-join DELETED_USERS as u on u.user_id = m.user_id
-left join DELETED_EMAILS as e on e.email = m.email
-where e.email is null;
-"""
+
+@task()
+def query_file(file_name: str, **kwargs):
```

Comment:
```suggestion
```
**Optional:** This _does_ work, but it shouldn't be necessary to wrap Prefect tasks in a task. To be more consistent with how we run tasks elsewhere, you could instantiate `SnowflakeQueriesFromFile` in this module without using a function.

Reply:
I now get the comment here, and this makes sense. Without using the function, we would've had to repeat setting the `account` and `user` parameters on every call.